server/src/controllers/leaderboardController.js

```diff
@@ -20,9 +21,9 @@ const addScore = async (req, res) => {
 };
 
 const getTopLeaders = async (req, res) => {
-  const topFive = await Leaderboard.find().sort('-score').limit(5);
+  const topLeaders = await Leaderboard.find().sort('-score').limit(LEADERBOARD_LIMIT);
```

Comment:
Suggestion you can take limit from client as query parameter.. like, 


'/api/leaderboard?limit=5'

then get the value on server using `req.query.limit`

And if no limit is provided, use default value.

Reply:
And on the client, you can actually pass params as an argument in the getAPI function.

```js
const getTopFive = getAPI('/api/leaderboard')({params: { limit: 5 }})
```

Reply:
I thought about this, and I thought we would need to have the LEADERBOARD_LIMIT variable in both environments, client and server, because we don't want to hardcode the limit. So I just kept it on the server.

Reply:
yes, but on the client it doesn't necessarily need to be a global constant.. its based on the component, 

hypothetically, say we have a mini-leaderboard which only displays 5 entries.. and we can bigger leaderboard displays 50 entries, both components have its own limit variable. 

This way the code is more extensible. I'll tag @nimish-gupta , I think he probably has even better idea on this.

Reply:
Okay, I see the benefit now. I will make this change.